Build on Mac fails after PR #50

See:
+ https://github.com/titsuki/raku-Chart-Gnuplot/pull/50
+ https://github.com/titsuki/raku-Chart-Gnuplot/actions/runs/18242691283/job/51946666295


Maybe 6.0.3 has a different directory structure or something download trouble happens:
```
Run zef build .
===> Building: Chart::Gnuplot:ver<0.0.21>
Aborting due to build failure: Chart::Gnuplot:ver<0.0.21> (use --force-build to override)
[Chart::Gnuplot] Failed to change the working directory to '/Users/runner/work/raku-Chart-Gnuplot/raku-Chart-Gnuplot/gnuplot-6.0.3': does not exist
[Chart::Gnuplot]   in method install-gnuplot at /Users/runner/work/raku-Chart-Gnuplot/raku-Chart-Gnuplot/lib/Chart/Gnuplot/CustomBuilder.pm6 (Chart::Gnuplot::CustomBuilder) line 43
[Chart::Gnuplot]   in method build at /Users/runner/work/raku-Chart-Gnuplot/raku-Chart-Gnuplot/lib/Chart/Gnuplot/CustomBuilder.pm6 (Chart::Gnuplot::CustomBuilder) line 20
[Chart::Gnuplot]   in block <unit> at -e line 1
===> Building [FAIL]: Chart::Gnuplot:ver<0.0.21>
Error: Process completed with exit code 1.
```

--- a/lib/Chart/Gnuplot/CustomBuilder.pm6
+++ b/lib/Chart/Gnuplot/CustomBuilder.pm6
@@ -29,7 +29,7 @@ class Chart::Gnuplot::CustomBuilder:ver<0.0.21> is Distribution::Builder::MakeFr
             { module => "Zef::Service::Shell::curl" },
         ];
         my $fetcher      = Zef::Fetch.new(:backends(@fetch-backends));
-        my $uri          = 'https://fossies.org/linux/misc/gnuplot-6.0.3.tar.gz';
+        my $uri          = 'https://downloads.sourceforge.net/project/gnuplot/gnuplot/6.0.3/gnuplot-6.0.3.tar.gz';
         my $archive-file = "gnuplot-6.0.3.tar.gz".IO.e
         ?? "gnuplot-6.0.3.tar.gz"
         !! $fetcher.fetch(Candidate.new(:$uri), "gnuplot-6.0.3.tar.gz");

